docs/architecture/blazor-for-web-forms-developers/data.md

```diff
@@ -19,7 +19,7 @@ Data Sources were controls that you could place on a Web Forms page and configur
 
 ![Data Sources](media/data/datasources.png)
 
-ADO.NET is the low-level approach to interacting with a database. Your apps could create a connection to the database with Commands, Recordsets, and Datasets for interacting. The results could then be bound to fields on screen without much code. The drawback of this approach was that each set of ADO.NET objects (`Connection`, `Command`, and `Recordset`) was bound to libraries provided by a database vendor. Use of these components made the code rigid and difficult to migrate to a different database.
+ADO.NET is the low-level approach to interacting with a database. Your apps could create a connection to the database with Commands, Recordsets, and Datasets for interacting. The results could then be bound to fields on screen without much code. The drawback of this approach was that each set of ADO.NET objects (`Connection`, `Command`, and `Recordset`) was bound to libraries provided by a database vendor. Use of these components, made the code rigid and difficult to migrate to a different database.
```

Comment:
I think the added comma is extra here.

Reply:
Hey @Youssef1313 , wanted to represent "Use of these components" as a single phrase in that sentence and that's the reason thought of adding that comma.

Reply:
@sughosneo I'm not sure (probably I'm wrong). I'll defer to @gewarren on whether there should be a comma or not.

Reply:
drop the comma..

Reply:
Thanks @Youssef1313 and @nishanil.  Done